### PR TITLE
Fixes for CI/CD compatibility

### PR DESCRIPTION
**Pull request by rishic3**
### Changes: CI/CD
1. Avoid hardcoding RAPIDS Jar version for CI/CD. Instead, let pipeline download and store jar path in $RAPIDS_JAR environment variable, as specified in [this PR](https://gitlab-master.nvidia.com/nvspark/blossom-jenkins/-/merge_requests/1858). Allows us to pull the appropriate plugin version from Spark-RAPIDS. 
2. Use unique (timestamped) paths for the TensorRT timing cache, in response to [this bug](https://nvbugswb.nvidia.com/NvBugs5/ISBug.aspx?bugid=4913642&cmtNo=), so that cached files can be properly cleaned up. 

### Bug Fixes:
- Deleted duplicate image_classification notebook in Tensorflow

### CODE REVIEW

File: examples/ML+DL-Examples/Spark-Rapids-ML/pca/notebooks/pca.ipynb [hunk 1]
```diff
@@ -119,7 +98,7 @@
     "# Check if Spark session is already active, if not, initialize it\n",
     "if 'spark' not in globals():\n",
     "    print(\"No active Spark session found, initializing manually.\")\n",
-    "    rapids_jar = get_rapids_jar()\n",
+    "    rapids_jar = os.environ['RAPIDS_JAR']\n",
```
Comment (jlowe) [inline examples/ML+DL-Examples/Spark-Rapids-ML/pca/notebooks/pca.ipynb:101]: This change seems to make the notebook less turnkey for users who want to run it as an example.  Now the notebook requires the user to setup the RAPIDS_JAR environment variable where the kernel runs, and the notebook won't run properly at all until that is done.

How about changing the notebook so it works as it did before if RAPIDS_JAR is _not_ set in the kernel environment, but allows the jar location to be overridden if it is set?

Comment (rishic3) [reply examples/ML+DL-Examples/Spark-Rapids-ML/pca/notebooks/pca.ipynb:101]: We instruct users to use [Pyspark to launch juptyer](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-24.10/examples/ML%2BDL-Examples/Spark-Rapids-ML/pca#running-the-notebooks), so this part of the code is really just for the CI pipeline, which requires manual initialization of a Spark session. 

That said, your comment might be more relevant to the [startup script](https://github.com/NVIDIA/spark-rapids-examples/blob/branch-24.10/examples/ML%2BDL-Examples/Spark-Rapids-ML/pca/start-spark-rapids.sh). Will make this fix.

**Review by jlowe — DISMISSED** [review]
This looks OK to me.

**Review by eordentlich — DISMISSED** [review]
👍

**Review by YanxuanLiu — DISMISSED** [review]
LGTM for PCA part. We can remove the DL changes, since we discussed with @rishic3 on Slack that we will run dl notebook pipeline in container, we do not need to care about the cache dirs.

**Comment by rishic3** [comment]
Markdown link checker is unhappy with [tensorflow links](https://www.tensorflow.org/tutorials/) for an unknown reason. Switching to use links to the Github source instead for now.

**Review by eordentlich — APPROVED** [review]
👍

**Pull request merged**